backend/coreapp/models/project.py

```diff
@@ -41,7 +43,10 @@ def import_functions(self) -> None:
             import_config.execute_import()
 
     def is_member(self, profile: Profile) -> bool:
-        return ProjectMember.objects.filter(project=self, profile=profile).exists()
+        for member in self.members():
```

Comment:
This should stay as a filter imo - it's much more Django-ish

Reply:
IIRC this broke tests, but I can revisit

Reply:
Breaks tests so leaving as-is (profile.user is mocked and explodes)